news/2022-08-04-wiki-contributors-wanted.md

```diff
@@ -0,0 +1,25 @@
+---
+layout: post
+title: "Wiki contributors wanted!"
+date: 2022-08-04 10:00:00 +0000
+---
+
+Lore maestros! Technical aficionados! Information brokers! We seek any kind of knowledge to share in the wiki, and to translate it for every corner of the community.
```

Comment:
```suggestion
Lore maestros! Technical aficionados! Information brokers! We seek to share any kind of knowledge in the wiki, and to translate it for every corner of the community.
```
some parallelism with "we seek [to share | to translate]"

Reply:
very cool